initial commit

AleksandrPok opened the pull request.

AleksandrPok committed:
initial commit

Max-Pochepets commented on src/main/java/cinema/model/dto/OrderResponseDto.java:
```diff
@@ -0,0 +1,42 @@
+
+public class OrderResponseDto {
+    private Long id;
+    private List<Long> tickets;
```
```suggestion
    private List<Long> ticketIds;
```

Max-Pochepets commented on src/main/java/cinema/model/dto/ShoppingCartResponseDto.java:
```diff
@@ -0,0 +1,24 @@
+
+public class ShoppingCartResponseDto {
+    private Long userId;
+    private List<Long> tickets;
```
same

KseniiaMakarova commented on src/main/java/cinema/controller/AuthenticationController.java:
```diff
@@ -0,0 +1,23 @@
+    }
+
+    @PostMapping("/register")
+    public void registration(@RequestBody UserRequestDto userRequestDto) {
```
```suggestion
    public void register(@RequestBody UserRequestDto userRequestDto) {
```
method names should be verbs

KseniiaMakarova commented on src/main/java/cinema/controller/ShoppingCartController.java:
```diff
@@ -0,0 +1,44 @@
+    }
+
+    @PostMapping("/movie-session")
+    public void add(@RequestParam Long userId, @RequestParam Long movieSessionId) {
```
```suggestion
    public void addSession(@RequestParam Long userId, @RequestParam Long movieSessionId) {
```

KseniiaMakarova commented on src/main/java/cinema/service/mappers/UserMapper.java:
```diff
@@ -0,0 +1,23 @@
+
+@Component
+public class UserMapper {
+    public User mapToEntity(UserRequestDto userRequestDto) {
```
do we need this method?

AleksandrPok committed:
refactored code:
renamed method registration to register in AuthenticationController class;
renamed method add to addSession in ShoppingCartController class;
deleted unnecessary method in UserMapper class
